Schema/pullRequest.schema.json

```diff
@@ -6,45 +6,33 @@
     "type": "object",
     "properties": {
         "merged_at": {
-            "type": [
-                "string",
-                "null"
-            ],
-            "format": "date-time"
+            "type": "string",
+            "format": "date-time",
+            "nullable": true
         },
         "diff_url": {
-            "type": [
-                "string",
-                "null"
-            ],
-            "format": "uri"
+            "type": "string",
+            "format": "url",
+            "nullable": true,
```

Comment:
We discussed this one, technically possible, but not doable. Please change it back.